### PR TITLE
Updated Combination graph datasets and examples to maintain consistency.

### LINKED ISSUE #207: Update Combination graph example datasets and reflow logic to maintain consistency
# Story
JIRA: [UXPLATFORM-5482](https://jira2.cerner.com/browse/UXPLATFORM-5482)
## Description

As a developer, I would like to make changes to the example datasets and example reflow logic for combination graphs.

## Motivation

So that consistency can be maintained across all the examples present in the terra-dev site.

## Definition of Done

We'll know we're done when the datasets and reflow logic for combination examples are consistent:

- Reflow logic for both react and carbon-centric examples are updated to be consistent with this logic: https://github.com/cerner/terra-graphs/pull/193#discussion_r679991504 
- All datasets are updated to be non arrays	
- Dedicated graph config objects are added for the examples where they are dynamically updated

### PR DESCRIPTION
**Pull request by AshishMotanamGurunadham**
### Summary

- Updated Combination graph config objects and datasets in examples to maintain consistency.

Closes #207 

### Deployment Link

https://terra-graphs-deployed-pr-#.herokuapp.com/

Thank you for contributing to Terra.
@cerner/terra
@cerner/carbon

### CODE REVIEW

**Comment by sdadn** [comment]
Please rebase with main and resolve conflicts

File: packages/terra-graphs-docs/src/example-datasets/dataObjects/Combination/lineSimpleRegionDataset.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import {
+  loadPopup,
```
Comment (sdadn) [inline packages/terra-graphs-docs/src/example-datasets/dataObjects/Combination/lineSimpleRegionDataset.js:3]: nit: remove **,** to make it a single line

**Review by tbiethman — APPROVED** [review]

File: packages/terra-graphs-docs/src/example-datasets/dataObjects/Combination/lineSimpleRegionDataset.js (hunk 1 above)
Comment (AshishMotanamGurunadham) [reply packages/terra-graphs-docs/src/example-datasets/dataObjects/Combination/lineSimpleRegionDataset.js]: Fixed [here](https://github.com/cerner/terra-graphs/pull/235/commits/2693d1695999667f0eaee02713222567d35784ed).

File: packages/terra-graphs-docs/src/terra-dev-site/test/graphs/Combination/BasicCombinationGraphs/LineWithMultipleBubbleData.test.jsx [hunk 2]
```diff
@@ -3,10 +3,12 @@ import utils from '@cerner/carbon-graphs/lib/js/helpers/utils';
...
 import '@cerner/terra-graphs-docs/lib/terra-dev-site/ExampleGraphContainer/ExampleGraphContainer.module.scss';
 import getCombinationGraphConfig from '@cerner/terra-graphs-docs/lib/example-datasets/graphConfigObjects/Combination/combinationDefault';
-import data from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/lineWithMultipleBubbleData';
+import lineDataset from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/lineDataset1';
+import multipleBubbleDataset1 from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/bubbleDataset1';
+import multipleBubbleDataset2 from '@cerner/terra-graphs-docs/lib/example-datasets/dataObjects/Combination/bubbleDataset2';
 
 const graphConfig = utils.deepClone(getCombinationGraphConfig('#lineWithMultipleBubbleData'));
-const dataset = utils.deepClone(data);
+const dataset = [utils.deepClone(multipleBubbleDataset1), utils.deepClone(multipleBubbleDataset2), utils.deepClone(lineDataset)];
```
Comment (sdadn) [inline packages/terra-graphs-docs/src/terra-dev-site/test/graphs/Combination/BasicCombinationGraphs/LineWithMultipleBubbleData.test.jsx:11]: nit for easier reading:
```suggestion
const dataset = [
                  utils.deepClone(multipleBubbleDataset1), 
                  utils.deepClone(multipleBubbleDataset2), 
                  utils.deepClone(lineDataset)
                ];
```

**Review by ryanthemanuel — APPROVED** [review]

**Review by sdadn — APPROVED** [review]

**Pull request merged**